morph/Visual.h

```diff
@@ -255,6 +258,30 @@ namespace morph {
         // A callback friendly wrapper
         static void set_context (morph::Visual<glver>* _v) { _v->setContext(); };
 
+        //! Lock the context mutext to prevent locking across multiple. Then sets the context.
+        void lockContext() { 
+            context_mutex.lock(); 
```

Comment:
this->context_mutex and this->setContext

Reply:
updated